### ISSUE TITLE
Split disjunctions in coverage analysis

### ISSUE THREAD

**Issue by senier** [Componolit/RecordFlux-specifications#31]
If a link condition is a disjunction, we consider it covered if any part of this disjunction is true. For more realistic test cases, we should split those disjunctions and treat them separately in the coverage analysis (i.e. every part of the disjunction must be true).

We should discuss whether it should be possible to disable/enable this feature through a command line option.

**Comment by treiher** [comment]
I agree, splitting disjunctions makes sense. I think there is no need for a command line option as long as we have no use case where would like to disable that feature.

**Comment by senier** [comment]
> I think there is no need for a command line option as long as we have no use case where we would like to disable that feature.

I could imagine that you would want to disable this feature initially to reach 100% link coverage and later enable it (IMHO the difference in required test cases will be very significant). But we can add it later if needed.

**Comment by jklmnn** [comment]
There is already code implementing the splitting of disjunctions into lists in [RecordFuzz/rfzz/rflx/transform.py:677](https://github.com/Componolit/RecordFuzz/blob/d54427bb56ed6d783385ded89e41a99080c26390/rfzz/rflx/transform.py#L677). This code might be reused.
